src/models/subscription.ts

```diff
@@ -25,6 +25,28 @@ export interface Repository {
 	updated_at: string; // TODO: is this a date object or a timestamp?  Different places uses different things
 }
 
+export interface Branch {
```

Comment:
Just curious where the schema is it coming from, and where it is used?

Reply:
Yes this is generated based on its usage

Reply:
I'm wondering if that's the right place for this interface? Don't think we ever store branch data in our Db...

Reply:
Happy to move it if you have a better location for it

Reply:
maybe to `src/github/client/github-client.types.ts` ?

Reply:
I see you have copied it to github-client.types... shall we remove this one then?